app/src/components/header.js

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Link, StaticQuery, graphql } from "gatsby"
+import Navbar from "react-bootstrap/Navbar"
+import Nav from "react-bootstrap/Nav"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faHome, faQuestionCircle } from "@fortawesome/free-solid-svg-icons"
+import "bootstrap/dist/css/bootstrap.min.css"
+
+
+const NavItem = props => (
+  <div className="nav-item" style={{ fontSize: '15px', margin: '10px' }}>
+    <p style={{ textAlign: 'left', marginTop: '10px' }}>
+      <Link to={props.href}>
+        <FontAwesomeIcon icon={props.icon} />&nbsp;
+        <strong>{props.title}</strong>
+      </Link>
+    </p>
+  </div>
+)
+
+const Header = ({ siteTitle }) => (
+  <StaticQuery
+    query={graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+            description
+            author
+          }
+        }
+      }
+    `}
+    render={data => (
+      <Navbar collapseOnSelect expand="lg" bg="light" variant="light">
```

Comment:
Here you can see some of the react-bootstrap components at work (in this case, `<Navbar>`).